bitnami/minio/templates/_helpers.tpl

```diff
@@ -97,6 +97,52 @@ imagePullSecrets:
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return MinIO accessKey
+*/}}
+{{- define "minio.accessKey" -}}
+{{- if .Values.global.minio.accessKey }}
+    {{- if .Values.global.minio.accessKey.password }}
+        {{- .Values.global.minio.accessKey.password -}}
+    {{- else if (not .Values.global.minio.accessKey.forcePassword) }}
+        {{- randAlphaNum 10 -}}
+    {{- else -}}
+        {{ required "An Access Key is required!" .Values.global.minio.accessKey.password }}
+    {{- end -}}
+{{- else if .Values.accessKey -}}
+    {{- if .Values.accessKey.password }}
+        {{- .Values.accessKey.password -}}
+    {{- else if (not .Values.accessKey.forcePassword) }}
+        {{- randAlphaNum 10 -}}
+    {{- else -}}
+        {{ required "An Access Key is required!" .Values.accessKey.password }}
+    {{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return MinIO secretKey
+*/}}
+{{- define "minio.secretKey" -}}
+{{- if .Values.global.minio.secretKey }}
+    {{- if .Values.global.minio.secretKey.password }}
+        {{- .Values.global.minio.secretKey.password -}}
+    {{- else if (not .Values.global.minio.secretKey.forcePassword) }}
```

Comment:
It is not clear to me why you would want the forcePassword parameter to be accessible in other subcharts

Reply:
So you suggest sth like this:
```
{{/*
Return MinIO secretKey
*/}}
{{- define "minio.secretKey" -}}
{{- if .Values.global.minio.secretKey }}
    {{- if .Values.global.minio.secretKey.password }}
        {{- .Values.global.minio.secretKey.password -}}
    {{- else if (not .Values.secretKey.forcePassword) }}
        {{- randAlphaNum 40 -}}
    {{- else -}}
        {{ required "A Secret Key is required!" .Values.global.minio.secretKey.password }}
    {{- end -}}
{{- else if .Values.secretKey -}}
    {{- if .Values.secretKey.password }}
        {{- .Values.secretKey.password -}}
    {{- else if (not .Values.secretKey.forcePassword) }}
        {{- randAlphaNum 40 -}}
    {{- else -}}
        {{ required "A Secret Key is required!" .Values.secretKey.password }}
    {{- end -}}
{{- end -}}
{{- end -}}
```

Reply:
???

Reply:
Yes